src/s2wasm.h

```diff
@@ -665,22 +668,31 @@ class S2WasmBuilder {
 
     mustMatch(":");
 
+    Function::DebugLocation debugLocation = {0,0,0};
```

Comment:
here and in a few more places, spaces after `{` and before `}` would be more consistent with the rest of the code